qemu/tests/cfg/block_commit.cfg

```diff
@@ -18,6 +18,22 @@
     base_image = "images/sn1"
     top_image = "images/sn3"
     expected_image_file = "images/sn1"
+    Windows:
+        create_command = "WIN_UTILS:\coreutils\DummyCMD.exe FILE 1073741824 1"
+        file_names = "C:\live_commit_test"
+        test_exists_cmd = "dir FILE"
+        autostress = no
+        start_cmd = '"${install_path}\heavyload.exe" \CPU 1 \MEMORY 1024'
+        x86_64:
+            sync_cmd = "WIN_UTILS:\Sync\sync64.exe /accepteula"
+        i386, i686:
+            sync_cmd = "WIN_UTILS:\Sync\sync.exe /accepteula"
+    Linux:
+        create_command = "dd if=/dev/urandom of=FILE bs=128k count=100"
+        file_names = "/home/live_commit_test"
+        test_exists_cmd = "test -f FILE"
+        control_args = "--cpu 1 --vm 1 --vm-bytes 16M"
+        sync_cmd = "sync"
```

Comment:
Why add L21-L36 in commit 1da9598d73f8d6d9233d850cba85bb645471b21f (add test case for general operation) ?
From my checking,  it is not used in case general_operation.